chainer/functions/loss/mean_absolute_error.py

```diff
@@ -42,7 +42,12 @@ def mean_absolute_error(x0, x1):
     """Mean absolute error function.
 
     This function computes mean absolute error between two variables. The mean
-    is taken over the minibatch.
+    is taken over the minibatch. Args x0 and x1 must be of the same dimension!
```

Comment:
I agree that the constraint on the dimension is somewhat surprising, especially after #4679.  Nevertheless I'd rather not put `!` because the constraint is a convention of loss functions in Chainer.